src/content/en/updates/2018/10/devtools.md

```diff
@@ -0,0 +1,277 @@
+
+<figure>
+  <img src="/web/updates/images/2018/10/focus4.png"
+       alt="The blue border remains even though the page doesn't really have focus."/>
+  <figcaption>
+    <b>Figure 7</b>. The blue border remains even though the page doesn't really have focus
+  </figcaption>
+</figure>
+
+## Initiator and priority information now in HAR imports and exports {: #HAR }
+
+If you'd like to diagnose network logs with colleagues, you can [export the network requests
+to a HAR file][export].
+
+[export]: /web/tools/chrome-devtools/network-performance/reference#save-as-har
+
+<figure>
+  <img src="/web/tools/chrome-devtools/network-performance/imgs/save-as-har.png"
+       alt="Exporting network requests to a HAR file."/>
+  <figcaption>
+    <b>Figure 8</b>. Exporting network requests to a HAR file
+  </figcaption>
+</figure>
+
+To import the file back into the Network panel, just drag and drop it.
+
+When you export a HAR file, DevTools now includes initiator and priority information in
+the HAR file. When you import HAR files back into DevTools, the **Initiator** and **Priority** columns
+are now populated.
+
+The `_initiator` field provides more context around what caused the resource to be requested.
+This maps to the **Initiator** column in the Requests table.
+
+<figure>
+  <img src="/web/updates/images/2018/10/HAR1.png"
+       alt="The initiator column."/>
+  <figcaption>
+    <b>Figure 9</b>. The initiator column
+  </figcaption>
+</figure>
+
+You can also [hold <kbd>Shift</kbd> and hover over a request][initiator] to view its initiator and dependencies.
+
+[initiator]: /web/tools/chrome-devtools/network-performance/reference#initiators-dependencies
+
+<figure>
+  <img src="/web/tools/chrome-devtools/network-performance/imgs/initiators-dependencies.png"
+       alt="Viewing initiators and dependencies."/>
+  <figcaption>
+    <b>Figure 10</b>. Viewing initiators and dependencies
+  </figcaption>
+</figure>
+
+The `_priority` field states what priority level the browser assigned to the resource. This maps to the
+**Priority** column in the Requests table, which is hidden by default.
+
+<figure>
+  <img src="/web/updates/images/2018/10/HAR2.png"
+       alt="The Priority column."/>
+  <figcaption>
+    <b>Figure 11</b>. The Priority column
+  </figcaption>
+</figure>
+
+Right-click the header of the Requests table and select **Priority** to show the **Priority** column.
+
+<figure>
+  <img src="/web/updates/images/2018/10/HAR3.png"
+       alt="How to show the Priority column."/>
+  <figcaption>
+    <b>Figure 12</b>. How to show the <b>Priority</b> column
+  </figcaption>
+</figure>
+
+Note: The `_initiator` and `_priority` fields begin with underscores because the [HAR spec][spec]{: .external }
+states that custom fields must begin with underscores.
+
+[spec]: https://dvcs.w3.org/hg/webperf/raw-file/tip/specs/HAR/Overview.html
+
+## Access the Command Menu from the Main Menu {: #command-menu }
```

Comment:
We should also let people know about the keyboard shortcut that already existed. Eg, something like "The Command Menu provides a list of all available Devtools panels, tabs, and features. It used to only be accessible by typing  ..., and not everyone knew the keyboard shortcut. Now you can access it directly from the Main Menu." Or more conversationally, we could say, "Did you know about the [...] keyboard shortcut? It shows the Command Menu, a list of... Now you don't need to know the shortcut. The Command Menu appears in the Main Menu."